### PR TITLE
chore: align Docker files between OS and ENT flavours

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: None
Ticket: QA-733

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --fast

File: worker/data/.gitkeep
Comment (merlin-northern) [inline worker/data/.gitkeep:1]: is this also needed?

File: Dockerfile.worker [hunk 1]
```diff
@@ -20,7 +20,9 @@ EXPOSE 8080
 COPY --from=builder /etc_extra/ /etc/
 USER 65534
 COPY --from=builder --chown=nobody /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder --chown=nobody /tmp /tmp
```
Comment (merlin-northern) [inline Dockerfile.worker:23]: whats in the tmp in the builder?

**Review by merlin-northern — APPROVED** [review]
approving not to block left some comments where I have doubts.

File: worker/data/.gitkeep
Comment (tranchitella) [reply worker/data/.gitkeep:1]: It is needed to keep the folder there in git, I believe.

File: Dockerfile.worker (hunk 1 above)
Comment (tranchitella) [reply Dockerfile.worker:23]: It is the "standard" tmp folder, which is used by the worker to process things (in enterprise).

**Pull request merged**